dist/modules/MainScreen/styles.js

```diff
@@ -0,0 +1,13 @@
+import styled from 'styled-components';
+export const MainWrapper = styled.View `
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
+`;
+export const MainTitle = styled.Text `
```

Comment:
same here a lot of duplicated styles, there are few styled component which is doing the same.